ts/localredirectiondebugger/v1.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Supports working with the Debug (Redirect locally) command.
+ */
+export interface LocalRedirectionDebuggerV1 {
+    /**
+     * Registers a local redirection debugger. The debugger will be invoked on user action
+     * when the user selects the Debug (Redirect locally) command in the cluster explorer,
+     * or when running the Debug (Redirect locally) command in the command palette. The
+     * local redirection debugger wil show its own user interface once invoked.
+     * @param localRedirectionDebugger The local redirection debugger to be registered.
+     */
+    register(localRedirectionDebugger: LocalRedirectionDebuggerV1.LocalRedirectionDebugger): void;
+    /**
+     * Lists registered local redirection debuggers.
+     * @returns The local redirection debuggers registered for the Debug (Redirect locally) command.
+     */
+    list(): ReadonlyArray<LocalRedirectionDebuggerV1.LocalRedirectionDebugger>;
+}
+
+export namespace LocalRedirectionDebuggerV1 {
+    /**
+     * Represents a local redirection debugger - an object that provides a
+     * user interface for working with a type of debugger redirecting the
+     * traffic from the user's cluster to the local machine.
+     */
+    export interface LocalRedirectionDebugger {
+        /**
+         * A programmatic identifier for the local redirection debugger.
+         */
+        readonly id: string;
+        /**
+         * Invoked by the Kubernetes extension on user action to start
+         * the debugger on the targeted node.
+         * @param target The object passed by Visual Studio Code as the
+         * command target. Use the cluster explorer API to resolve this
+         * target (resolveCommandTarget method). This parameter will be
+         * undefined if the debugging action is initiated through the
+         * Visual Studio command palette.
+         */
+        startDebugging(target?: any): void;
```

Comment:
As you can see, I kept the interface simple enough as here we just want to let the local redirection debugger know that it has to start debugging on a specific target (or no target if invoked from the command palette).
I chose to use a `target: any` parameter and let the extension take care of calling `resolveCommandTarget`, because resolving the command target and providing directly a `CloudExplorerNode` would seem to me like breaking the versions mechanism you created.
That said, if you think this would be better and that taking dependencies cross APIs is OK, please let me know.